spec/workflow/states/wait_spec.rb

```diff
@@ -73,10 +99,23 @@
         end
         expect(state.running?(ctx)).to be_falsey
       end
+
+      it "runs" do
+        workflow.run_nonblock
+        expect(workflow.end?).to eq(false)
+        expect(workflow.context.state_history.size).to eq(0)
+
+        Timecop.travel(Time.now.utc + 100) do
+          workflow.run_nonblock
+        end
+
+        expect(workflow.end?).to eq(true)
+        expect(workflow.context.state_history.size).to eq(2)
+      end
```

Comment:
These other contexts didn't have the `it "runs"` like TimestampPath did.

Reply:
total aside, you can also just do `it do` without any text.

Reply:
I've started really liking `it do` - especially when we have it runs / it throws an exception

Reply:
Or I'll just give it a more descriptive name